docs/CloudSpanner.md

```diff
@@ -0,0 +1,64 @@
+Trillian on Cloud Kubernetes with Cloud Spanner
+====================================================
+
+This document outlines the steps necessary to get an instance of Trillian up
+and running on Google Cloud.
+
+ 1. Create a Google Cloud project (e.g. trillian-opensource-ci)
```

Comment:
Is that our project name? I think they have to be unique so might be better to generificate it.

Reply:
done